src/components/shared/MarkerContextMenu.js

```diff
@@ -384,15 +384,22 @@ class MarkerContextMenuImpl extends PureComponent<Props> {
         {data && data.type === 'IPC' ? (
           <MenuItem onClick={this.selectOtherThreadForIPCMarkers}>
             <span className="react-contextmenu-icon markerContextMenuSelectThread" />
-            {data.direction === 'sending' ? (
-              <Localized id="MarkerContextMenu--select-the-receiver-thread">
-                Select the receiver thread
+            {data.direction === 'sending' && data.recvTid ? (
+              <Localized
+                id="MarkerContextMenu--select-the-receiver-thread"
+                vars={{ threadName: data.recvThreadName }}
+              >
+                <>Select the receiver “{data.recvThreadName}” thread</>
```

Comment:
super optional nit: I was wondering if we could make the thread name bold, to highlight it more. Maybe with a text such as: `Select the receiver thread “<span>{data.recvThreadName}</span>“` 
Also I've moved the word "thread" before the name, it looks more natural to me.

As a reference this is how it looks like now:
![image](https://user-images.githubusercontent.com/454175/160129401-acbe91b7-8994-45af-9242-b971ba699379.png)


What do you think?